src/main/webapp/partials/instructorHelpSessions.jsp

```diff
@@ -6154,12 +6154,20 @@
                   </ul>
 
                   <br>
+                  <div class="col-sm-12 form-inline mobile-align-left">
+                    <label for="input" style="text-indent: 24px">
+                      <span>Evaluee/Recipient</span>
```

Comment:
For this, it should not just be a simple span. There should be a tooltip data-toggle as well. Although in the instructor help page the tool tips do not show up, it looks different. This is what you have done:

![image](https://user-images.githubusercontent.com/24493463/37245196-9e11f130-24cf-11e8-9b7f-27f32e2c57b0.png)

versus how an actual rubrics question looks like:

![image](https://user-images.githubusercontent.com/24493463/37245195-98fe664c-24cf-11e8-8515-a49c9b64ad1e.png)

You can also search for the term `Evaluee` in the same file to see how another question has done it.

Reply:
Yes that is actually what I am considering. Since it is an example showing in the instructor help page, i think it is not necessary to have the tooltip. But it does look different from the actual question display. I will work on it now.